sources/models/shared-defaults/aws-host-containers.toml

```diff
@@ -4,7 +4,7 @@ superpowered = true
 
 [metadata.settings.host-containers.admin.source]
 setting-generator = "schnauzer settings.host-containers.admin.source"
-template = "{{ ecr-prefix settings.aws.region }}/bottlerocket-admin:v0.8.0"
+template = "{{ ecr-prefix settings.aws.region }}/bottlerocket-admin:v0.9.0"
```

Comment:
nit: I'd prefer splitting the commit into two, one to move these defaults to the new host containers, and the other to add the migrations